EIPS/eip-3041.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 3041
+title: Adds `baseFee` to `eth_getBlockByHash`
+author: Abdelhamid Bakhta (@abdelhamidbakhta)
+discussions-to: https://ethereum-magicians.org/t/eip-3041-add-basefee-in-eth-getblockbyhash-response/4825
+status: Draft
+type: Standards Track
+category : Interface
+created: 2020-10-13
+requires: 1474, 1559
+---
+
+## Simple Summary
+Add basefee field to `eth_getBlockByHash` RPC endpoint response.
+
+## Abstract
+Adds `baseFee` property to the `eth_getBlockByHash` JSON-RPC request `result` object.  This property will contain the value of the base fee for any block after the EIP-1559 fork.
+
+## Motivation
+[EIP-1559](./eip-1559.md) introduces a base fee per gas in protocol.
+This value is maintained under consensus as a new field in the block header structure.
+Users may need value of the base fee at a given block. Base fee value is important to make gas price predictions more accurate.
+
+## Specification
+
+### `eth_getBlockByHash`
+
+#### Description
+
+Returns information about a block specified by hash.
+Every block returned by this endpoint whose block number is before the [EIP-1559](./eip-1559.md) fork block **MUST NOT** include a `baseFee` field.
+Every block returned by this endpoint whose block number is on or after the [EIP-1559](./eip-1559.md) fork block **MUST** include a `baseFee` field.
+
+#### Parameters
+
+Parameters remain unchanged.
+
+#### Returns
+For the full specification of `eth_getBlockByHash` see [EIP-1474](./eip-1474.md).
+Add a new JSON field to the `result` object for block headers containing a base fee (post [EIP-1559](./eip-1559.md) fork block).
+
+- {[`Quantity`](#quantity)} `baseFee` - base fee for this block
```

Comment:
```suggestion
- {[`Quantity`](./eip-1474.md#quantity)} `baseFee` - base fee for this block
```
This is being interpreted as a link `[title](link)` and it isn't a valid link because there is no `#quantity` anchor.  I'm guessing you copied this from 1474, so I think the fix here is to just change it to link to that other spec.